src/cpp/rtps/security/SecurityManager.cpp

```diff
@@ -192,6 +192,20 @@ bool SecurityManager::init(
             }
         }
 
+        const auto log_info_message = [this](const char* msg)
+                {
+                    if (logging_plugin_)
+                    {
+                        SecurityException logging_exception;
+                        logging_plugin_->log(LoggingLevel::INFORMATIONAL_LEVEL, msg, "SecurityManager,init",
+                                logging_exception);
+                    }
+                    else
+                    {
+                        logInfo(SECURITY, msg);
```

Comment:
```suggestion
                        EPROSIMA_LOG_INFO(SECURITY, msg);
```

Use the new log macros instead of the legacy ones.